src/styles/styles.css

```
@@ -85,11 +85,15 @@
   color: #a82058;
 }
 
-.multi-menu-wrapper.red .multi-menu-item.menu-option:hover {
+.multi-menu-wrapper.red .multi-menu-item:not(.locked).menu-option:hover {
   background-color: rgba(239, 76, 48, 0.15);
   color: #ef4c30;
 }
```

What do you think about using additive rules for this? Maybe its just personal preference but I find them simpler to reason about.

```suggestion
.multi-menu-wrapper.red .multi-menu-item.menu-option:hover {
  background-color: rgba(239, 76, 48, 0.15);
  color: #ef4c30;
}

.multi-menu-wrapper.red .multi-menu-item.locked.menu-option:hover {
  background-color: transparent;
}
```

---

I think these locked rules are more general, it'd be redundant to add the transparent background to all multi menu's themes